src/Rpc/NodeGuardService.cs

```diff
@@ -23,6 +23,9 @@ public interface INodeGuardService
         ServerCallContext context);
 
     Task<RequestWithdrawalResponse> RequestWithdrawal(RequestWithdrawalRequest request, ServerCallContext context);
+
+    Task<GetAvailableWalletsResponse>
+        GetAvailableWallets(GetAvailableWalletsRequest request, ServerCallContext context);
```

Comment:
nit: I'd get rid of the extra newline here. Doesn't seem like the line is too long anyway